pom.xml

```diff
@@ -148,8 +148,8 @@
   </dependencies>
 
   <properties>
-    <bioformats.version>5.7.3</bioformats.version>
-    <imagej1.version>1.48s</imagej1.version>
```

Comment:
We should be able to remove the ImageJ version for this repo

Reply:
Ah, very true, thanks.  I've pushed a commit to drop it.